tensorflow/python/ops/lookup_ops.py

```diff
@@ -1810,14 +1810,31 @@ def remove(self, keys, name=None):
 
     return op
 
-  def lookup(self, keys, name=None):
+  def lookup(self, keys, dynamic_default_values=None, name=None):
     """Looks up `keys` in a table, outputs the corresponding values.
 
     The `default_value` is used for keys not present in the table.
 
     Args:
       keys: Keys to look up. Can be a tensor of any shape. Must match the
         table's key_dtype.
+      dynamic_default_values: The values to use if a key is missing in the
+        table. If None (by default), the `self._default_value` will be used.
```

Comment:
table.default_value?

Reply:
I'm sorry, I can't get your point, maybe you mean what's the `self._default_value`?  I mean [here](https://github.com/tensorflow/tensorflow/blob/9343aeaec4caafa3e61089df01e68d68533d74d3/tensorflow/python/ops/lookup_ops.py#L1719  )

Reply:
Hi @rohan100jain , I just update the PR according to all other comments expect this point,  please review it again, Thanks!

Reply:
Oh this was just a minor documentation nit: Instead of saying self._default_value in the comment, just say table.default_value (users might not know the implementation details here)